components/org.wso2.carbon.identity.tools.saml.validator/src/main/java/org/wso2/carbon/identity/tools/saml/validator/processors/SAMLResponseBuilder.java

```diff
@@ -273,11 +273,12 @@ private AttributeStatement buildAttributeStatement(Map<String, String> claims) {
         AttributeStatement attStmt = null;
         if (claims != null) {
             attStmt = new AttributeStatementBuilder().buildObject();
-            Iterator<String> ite = claims.keySet().iterator();
+            Iterator<Map.Entry<String, String>> ite = claims.entrySet().iterator();
```

Comment:
We can simply get the entry set which is a 'Set'. Then we can use for each to process it. The existing code is not readable.